packages/infra/lib/api-stack/api-service.ts

```diff
@@ -488,7 +490,7 @@ export function createAPIService({
     maxCapacity: maxTaskCount,
   });
   scaling.scaleOnCpuUtilization("autoscale_cpu", {
-    targetUtilizationPercent: 80,
+    targetUtilizationPercent: 60,
```

Comment:
Prob won't make a diff, we prob need to get this down to 20-25% so auto scaling happens, as this is based on the cluster avg